modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -87,14 +88,46 @@ The index name must be unique among both indexes and constraints.
 ====
 
 A vector index is a single-label, single-property index for nodes.
-In addition to the label and property key (both given as `STRING`), a vector index needs to be configured with both the dimensionality of the vector (`INTEGER` between `1` and `2048` _inclusive_), and the measure of similarity between two vectors (case-insensitive `STRING`).
+A vector index needs to be configured with both the dimensionality of the vector (`INTEGER` between `1` and `2048` _inclusive_), and the measure of similarity between two vectors (case-insensitive `STRING`).
 For details, see xref:#indexes-vector-similarity[].
 
-.Signature for `db.index.vector.createNodeIndex` to create a vector node index
-[source,syntax,role="noheader",indent=0]
+[NOTE]
+====
+More details about the syntax descriptions can be found link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/syntax/#administration-syntax-reading[here].
+====
+
+.Syntax for creating vector indexes
+[options="header", width="100%", cols="5a, 3"]
+|===
+| Command | Description
+
+| [source, syntax, role=noplay, indent=0]
 ----
-db.index.vector.createNodeIndex(indexName :: STRING, label :: STRING, propertyKey :: STRING, vectorDimension :: INTEGER, vectorSimilarityFunction :: STRING)
+CREATE VECTOR INDEX [index_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+ON (n.propertyName)
+OPTIONS "{" option: value[, ...] "}"
 ----
+| Create a vector index on nodes.
+The options map is mandatory since there are mandatory index configuration settings, the vector dimensions and similarity function.
+
+|===
+
+It is considered best practice to give the index a name when it is created. This name is needed for both dropping and querying the index.
+If the index is not explicitly named, it will get an auto-generated name.
+
+[IMPORTANT]
+====
+The index name must be unique among all indexes and constraints.
+====
+
+The `CREATE VECTOR INDEX` command take an `OPTIONS` clause. This has two parts, the `indexProvider` and `indexConfig`.
+The provider can only have the default value, `'vector-1.0'`.
+The `indexConfig` is a map from string to string and integers, and is used to set the index-specific configuration settings, the vector dimensions and similarity function.
```

Comment:
Consider using an example instead, similar to [what we did for cdc](https://development.neo4j.dev/docs/cdc/current/procedures/output-schema/#_node_changes). That way the users can just copy paste the example & don't have to struggle to get the `[{"` in the right order.

Reply:
there is an example on creating the index further down though (line 172-177). 

I still think I want to keep the syntax descriptions and not only have examples.

I currently don't remember exactly how the restructuring of the sections will go but it might be that we have one page for all syntax descriptions (like we have for the search performance ones now) and not have fulltext and vector be separate 🤔